tools/backup/values.yaml

```diff
@@ -9,6 +9,7 @@ bucket: gs://test-neo4j
 database: neo4j,system
 # cloudProvider can be either gcp, aws, or azure
 cloudProvider: gcp
+# set secretName to NULL if using workload-identity in gcp
```

Comment:
This is the sort of thing you might want to document in the backup / restore docs (markdown), since otherwise you as the PR author are likely to be the only person who knows this is possible.